pkg/cmd/server/origin/legacy_test.go

```diff
@@ -0,0 +1,26 @@
+package origin
+
+import (
+	"testing"
+
+	"github.com/openshift/origin/pkg/api/latest"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func TestLegacyKinds(t *testing.T) {
+	for gvk := range api.Scheme.AllKnownTypes() {
+		if latest.OriginLegacyKind(gvk) {
```

Comment:
Actually, it appears that `OriginKind` is related to setting `/oapi` in nearly every usage.  We should remove it entirely.